tests/test_project.py

```diff
@@ -109,6 +111,51 @@ def test_default_model(self):
         assert len(project._models.data[0].sample) == 3
         assert len(project._materials) == 3
 
+    def test_sld_data_for_model_at_index(self):
+        # When
+        project = Project()
+        project.default_model()
+
+        # Then
+        sample_data = project.sld_data_for_model_at_index(0)
+
+        # Expect
+        assert len(sample_data.x) == 500
+        assert_almost_equal(
```

Comment:
It is preferred to use `assert_allclose`, which has improved features:

https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html

Reply:
changed assert in tests